taskcluster/tf_tc-build.sh

```diff
@@ -58,51 +14,32 @@ pushd ${DS_ROOT_TASK}/DeepSpeech/ds/tensorflow/
     # Force toolchain sync (useful on macOS ?)
     bazel ${BAZEL_OUTPUT_USER_ROOT} sync --configure
 
-    if [ "${build_amd64}" = "yes" ]; then
-        # Pure amd64 CPU-only build
-        if [ "${OS}" = "${TC_MSYS_VERSION}" -a  "${build_gpu}" = "no" ]; then
-            echo "" | TF_NEED_CUDA=0 ./configure && ${BAZEL_BUILD} -c opt ${BAZEL_OPT_FLAGS} ${BAZEL_EXTRA_FLAGS} ${BUILD_TARGET_LIB_CPP_API} ${BUILD_TARGET_LITE_LIB}
-        elif [ "${build_gpu}" = "no" -a "${build_linux_arm}" = "no" -a "${build_linux_arm64}" = "no" ]; then
-            echo "" | TF_NEED_CUDA=0 ./configure && ${BAZEL_BUILD} -c opt ${BAZEL_OPT_FLAGS} ${BAZEL_EXTRA_FLAGS} ${BUILD_TARGET_LIB_CPP_API} ${BUILD_TARGET_LITE_LIB}
-        fi
-
-        # Cross RPi3 CPU-only build
-        if [ "${build_linux_arm}" = "yes" ]; then
-            echo "" | TF_NEED_CUDA=0 ./configure && ${BAZEL_BUILD} -c opt ${BAZEL_ARM_FLAGS} ${BAZEL_EXTRA_FLAGS} ${BUILD_TARGET_LITE_LIB}
-        fi
-
-        # Cross ARM64 Cortex-A53 build
-        if [ "${build_linux_arm64}" = "yes" ]; then
-            echo "" | TF_NEED_CUDA=0 ./configure && ${BAZEL_BUILD} -c opt ${BAZEL_ARM64_FLAGS} ${BAZEL_EXTRA_FLAGS} ${BUILD_TARGET_LITE_LIB}
-        fi
-
-        # Pure amd64 GPU-enabled build
-        if [ "${build_gpu}" = "yes" ]; then
-            eval "export ${TF_CUDA_FLAGS}" && (echo "" | TF_NEED_CUDA=1 ./configure) && ${BAZEL_BUILD} -c opt ${BAZEL_CUDA_FLAGS} ${BAZEL_EXTRA_FLAGS} ${BAZEL_OPT_FLAGS} ${BUILD_TARGET_LIB_CPP_API}
-        fi
-    fi
-
-    if [ "${build_android_arm}" = "yes" ]; then
+    case "$1" in
+    "--cpu")
```

Comment:
Can you file a bug for later? I think it could be a good idea we decouple more here and have a list like: `"--linux-cpu"|"--windows-cpu"` etc. But that miight require quite some move love, so let's not hold your PR on that

Reply:
#3170